include/LadspaBase.h

```diff
@@ -35,30 +35,20 @@ namespace lmms
 
 class LadspaControl;
 
-
-typedef enum BufferRates
-{
+using buffer_rate_t = enum BufferRates {
```

Comment:
I am not sure why there was a ``typedef enum BufferRates`` and then it follows with ``buffer_rate_t`` at the end in the original code. I am assuming ``clang-tidy`` made no assumptions and kept both. I should be able to remove the ``enum BufferRates`` part without errors.

Reply:
Nevermind. While removing ``enum BufferRates`` doesn't produce errors, it gives warnings. Something like: ``warning: ‘lmms::PortDescription’ has a field ‘lmms::PortDescription::data_type’ whose type has no linkage [-Wsubobject-linkage]``. I'm guessing this is because an alias declaration is for aliasing other already named types? And since the enum is unnamed, it gives a linkage warning?

I think we can just declare ``enum buffer_rate_t`` as is, removing the using-declaration altogether.